lighthouse-extension/app/src/popup.js

```diff
@@ -134,10 +134,15 @@ document.addEventListener('DOMContentLoaded', _ => {
     })
     .catch(err => {
       let {message} = err;
```

Comment:
would you suggest changing it to `let message = err.message` or change message everywhere to `err.message`